omen2/codegen.py

```diff
@@ -108,13 +108,13 @@ def __init__(self, id, color: str = "green"):
             # we're in the init parameter line: `color: str`
             print(name_and_type, file=out, end="")
 
-            if col.default or not col.notnull:
+            if col.default is not None or not col.notnull:
                 # derive default value from the db default value
-                if col.default:
+                if col.default is not None:
                     try:
                         # check valid python
                         defval = pytype(str(col.default))  # pylint: disable=eval-used
-                    except (ValueError, NameError):
+                    except (ValueError, NameError, TypeError):
```

Comment:
when does this arise?

Reply:
if the default is some weird unexpected thing, like strftime("%s"), that doesn't translate to a python default

Reply:
ok traced it down.  in sqlite, a type can be different than a default.   you can have INT DEFAULT('yo').   we apparently rely on this, but i think it's ok to remove it and fix.

Reply:
removed, we'll see if we needed it

Reply:
yikes interesting. Okay sounds good